db/schema.rb

```diff
@@ -13,13 +13,6 @@
 
 ActiveRecord::Schema.define(version: 20160418150933) do
 
-  create_table "course_names", force: true do |t|
```

Comment:
You'll need to make sure and merge these changes in from master. This part shouldn't be in the diff.